src/SFML/Window/iOS/SFAppDelegate.mm

```diff
@@ -151,6 +162,36 @@ - (void)applicationWillTerminate:(UIApplication *)application
     }
 }
 
+- (bool)supportsOrientation:(UIDeviceOrientation)orientation
+{
+    if (!self.sfWindow)
```

Comment:
That would be nitpicking but I'd had some empty lines to make this function more readable, especially after return statements. What do you guys think?